src/js/rx/components/GlossaryModal.jsx

```diff
@@ -21,14 +21,12 @@ class GlossaryModal extends React.Component {
       element = (
         <div>
           <GlossaryList terms={this.props.content}/>
-          <div className="va-modal-button-group cf">
-            <button type="button" onClick={this.handleCloseModal}>Close</button>
-            <Link
-              to="/glossary"
-              onClick={this.props.onCloseModal}>
-              See all status definitions
-            </Link>
-          </div>
+          <button className="va-modal-close" type="button" onClick={this.handleCloseModal}><i className="fa fa-close"></i><span className="usa-sr-only">Close this modal</span></button>
+          <Link
```

Comment:
This is picky and you don't need to worry about changing it unless you want to, but when lines get real long like this it can be a little easier to read by breaking it into separate lines